modules/appendix/examples/ldbc_snb/select_statement_accum.gsql

```diff
@@ -3,9 +3,9 @@ CREATE QUERY accum_wrong_example() SYNTAX v2 {
     SumAccum<INT> @active_flag = 0;
 
     result = SELECT p
-             FROM Person: p - (KNOWS) - Person: w
+             FROM (p:Person) -[KNOWS]-> (w:Person)
```

Comment:
```suggestion
             FROM (p:Person) -[:KNOWS]- (w:Person)
```
1. Why did you put an arrow here? I look at there this example is used. I don't see any indication that this should be a directed edge.
2. You must have a colon before the edge type.